backend/src/main/java/cz/cuni/mff/xrg/odcs/backend/i18n/Messages.java

```diff
@@ -43,6 +44,7 @@ public static String getString(final String key, final Object... args) {
     private static ReloadableResourceBundleMessageSource initializeResourceBundle() {
         ReloadableResourceBundleMessageSource ms = new ReloadableResourceBundleMessageSource();
         ms.setDefaultEncoding("UTF-8");
+        ms.setFallbackToSystemLocale(false);
```

Comment:
Question: If it falls back to system locale, this should be only after it tries to get proper bundle based on settings of locale in config.properties, right?  So the question is how the situation in https://github.com/UnifiedViews/Core/issues/444 may happen? If Michal sets up "en" as default locale, fallback locale should never be used anyway (because default en locale is always available), am I right? So is it a bug in Spring? Or how it is possible that sk is used when en is set up in config.properties and available always?